Code/Tools/ProjectManager/Source/ProjectButtonWidget.h

```diff
@@ -88,11 +94,13 @@ namespace O3DE::ProjectManager
         Q_OBJECT
 
     public:
-        explicit ProjectButton(const ProjectInfo& m_projectInfo, QWidget* parent = nullptr);
-        ~ProjectButton() = default;
+        explicit ProjectButton(const ProjectInfo& projectInfo, const EngineInfo& engineInfo, QWidget* parent = nullptr);
```

Comment:
This function no longer needs to be explicit now that it accepts two parameters.
```suggestion
        ProjectButton(const ProjectInfo& projectInfo, const EngineInfo& engineInfo, QWidget* parent = nullptr);
```